src/components/chip-set/chip-set.scss

```diff
@@ -45,12 +50,6 @@ $leading-icon-space: 1.5rem;
     min-height: shared_input-select-picker.$height-of-mdc-text-field;
     position: relative;
 
-    &.chip-set--with-label {
```

Comment:
not particularly about this line, but could you take a look at the before and after:
before:
<img width="339" alt="Screenshot 2025-04-10 at 10 25 36" src="https://github.com/user-attachments/assets/358018db-2608-47ba-8555-fa1994379a59" />
after:
<img width="301" alt="Screenshot 2025-04-10 at 10 23 08" src="https://github.com/user-attachments/assets/df017179-f6bf-46b6-a42f-11e2cfde2a90" />

Do you want it to have it that close - label and chips? i like more with some space :)

Reply:
looks fine in web-client though. Let's have it as an improvement 🥰
![Screenshot 2025-04-10 at 13 55 38](https://github.com/user-attachments/assets/7ef25c83-b000-4e5b-ae8e-1920b80db083)

Reply:
This new one is the "correct" placement of it 😊. So in case a readonly with `type` of `input` (like in the web client) is placed side-by-side of a `type` of `choice` for example, then the alignment of their labels and content will be correct.